app/services/spree/product_feed_service.rb

```diff
@@ -71,11 +68,7 @@ def google_product_category
 
     def mpn
       mpn = property_value_by_name('MPN').to_s.downcase
-      if mpn.blank?
-        variant.sku
-      else
-        mpn
-      end
+      mpn.presence || variant.sku
```

Comment:
TIL about `presence` method.
https://guides.rubyonrails.org/active_support_core_extensions.html#presence

Reply:
WHOA. That's awesome.